modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/item/selector/FDSViewItemSelector.tsx

```diff
@@ -44,6 +44,10 @@ const FDSViewItemSelector = ({
 }) => {
 	const [selectedId, setSelectedId] = useState<String>();
 
+	const [externalReferenceCode, setSelectedExternalReferenceCode] = useState<
```

Comment:
We should follow the standard nomenclature here 
- `[selectedExternalReferenceCode, setSelectedExternalReferenceCode]` or 
- `[externalReferenceCode, setExternalReferenceCode]`

Reply:
oohh my bad!